datafusion/sqllogictest/test_files/joins.slt

```diff
@@ -2107,9 +2107,9 @@ RIGHT JOIN (select t2_id from join_t2 where join_t2.t2_id > 11) as join_t2
         ON join_t1.t1_id < join_t2.t2_id
 ----
 33 44
```

Comment:
this query has no ORDER BY so it is ok if the output emerges in a different order